src/en/authors-charm-store.md

```diff
@@ -108,13 +108,13 @@ entity: 0---1---2---3---4---5---6
 
 In doing so, the stable channel is updated to point to revision 3 and
 revision 3 is added to the channel history. The author can continue to
-push and publish. Since revisions are a constant stream there are
+push and release. Since revisions are a constant stream there are
 scenarios where the stable channel may be pointed to a higher revision
 even though development revision is actually newer.
 
 In the following example revision 8 is development, a bug is found in the
 latest stable revision (5) so a hot fix is applied and pushed as 9.
-That revision is then published to the stable channel, like this:
+That revision is then released to the stable channel, like this:
 
 ```
                                               stable (9, 5, 3, 2)
```

Comment:
The channels mentioned here are also out of date, but I think there is an issuee already covering that